src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: "What's new in Astro - December 2023"
+description: "December: last month of the year, a time to reflect on all the amazing things we've shipped, including Astro 4.0!"
+publishDate: "December 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-december-2023/og.webp"
+lang: "en"
+---
+
```

Comment:
I feel like linking to some of our launch week items does make sense in a monthly recap. If we’re including Docs on Starlight (which was a subitem of launch week) these are a big deal! To handle that, suggesting a small “News” section with these amongst others:

```suggestion

## News

- We launched a [$100,000 Astro Ecosystem Fund](https://astro.build/blog/astro-ecosystem-fund/) and would love for you to apply!
- Sentry is now Astro’s [Official Monitoring Partner](https://astro.build/blog/sentry-official-monitoring-partner/).
- Astro came top in user satisfaction in [Netlify’s State of Web Development](<https://www.netlify.com/resources/ebooks/the-state-of-web-development-2023/>) report — thanks everyone!
- [Astro Studio is coming soon…](https://twitter.com/astrodotbuild/status/1733199595231199584)

```

Reply:
I love this idea! I vote ##News becomes a new section!

Reply:
Oh yeah! I love this!

Great idea Chris!